components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -179,6 +180,14 @@ abstract class EngineSession(
      */
     abstract fun clearFindMatches()
 
+    /**
+     * Enables/disables fullscreen mode. It also invokes a callback when fullscreen mode is exited.
+     * For GeckoView, this shouldn't necessarily be used to
```

Comment:
The comment still mentions the callback :)

Reply:
Ah, sorry!